### ISSUE TITLE
Team ID and app Prefix mismatch

### ISSUE THREAD

**Issue by doapp-ryanp** [fastlane/fastlane#5904]
fastlane 1.101.0
PEM 1.3.2

I'm running into a strange situation where fastlane is reporting a different team id than what I see when I log into developer.apple.com

Fastlane (via PEM) is reporting that the Team ID my app is in is `EF6TA58Y38`.  When I log into developer.apple.com and go to `Identifiers > App IDs` 8 out of 10 of my apps list the prefix as `EF6TA58Y38`.  The other 2 list a completely different prefix of `B9VPQ9U3HT`.

How does fastlane choose which team ID to return in the list when prompting "Multiple teams found on the Developer Portal, please enter the number of the team you want to use"?

Or is this some sort of apple issue? 

Said another way, why would an app Prefix NOT match the Team ID that the app is in?

**Comment by doapp-ryanp** [comment]
Closing this ticket, just found out why there is a mismatch https://developer.apple.com/library/ios/technotes/tn2311/_index.html

tldr: old apps didnt use team id as their prefix

**Comment by TKBurner** [comment]
@doapp-ryanp Thanks for following up with those details. Hope you have a wonderful day! 👍
